src/wovnio/html/HtmlConverter.php

```diff
@@ -273,7 +273,14 @@ private function buildHrefLang($lang_code)
             $url = $this->headers->removeLang($url, $this->store->defaultLang());
             $url = Url::addLangCode($url, $this->store, $lang_code, $this->headers);
         } else {
-            if ($lang_code !== $this->store->defaultLang()) {
+            $customDomainLangs = $this->store->getCustomDomainLangs();
```

Comment:
I think it would be better to split this logic into a separate else-if "pattern is custom domain" block.